x-pack/plugins/fleet/server/services/agents/status.ts

```diff
@@ -84,28 +84,14 @@ export async function getAgentStatusForAgentPolicy(
   );
 
   return {
-    events: await getEventsCount(soClient, agentPolicyId),
     total: allActive.total,
     inactive: all.total - allActive.total,
     online: online.total,
     error: error.total,
     offline: offline.total,
     updating: updating.total,
     other: all.total - online.total - error.total - offline.total,
+    // Deprecated Agent events do not exists anymore
```

Comment:
Suggestion: Add jsdoc `@deprecated` in order to enable IDE features (normally a visual indicating the property is deprecated)

```suggestion
    /* @deprecated Agent events do not exists anymore */
```